flags/bitcode.cmake

```diff
@@ -7,12 +7,7 @@ else()
   set(POLLY_FLAGS_BITCODE_ 1)
 endif()
 
-include(polly_add_cache_flag)
+set(CMAKE_XCODE_ATTRIBUTE_ENABLE_BITCODE YES)
+set(CMAKE_XCODE_ATTRIBUTE_BITCODE_GENERATION_MODE "bitcode")
 
-polly_add_cache_flag(CMAKE_CXX_FLAGS "-fembed-bitcode")
-
-# We can't use 'ENABLE_BITCODE' because
-# it only adds '-fembed-bitcode-marker':
-# * https://stackoverflow.com/a/31346742
-
-list(APPEND HUNTER_TOOLCHAIN_UNDETECTABLE_ID "bitcode.2")
+list(APPEND HUNTER_TOOLCHAIN_UNDETECTABLE_ID "bitcode.2")
```

Comment:
"No newline at end of file" warning:
* http://0.readthedocs.io/en/latest/git.html#check-there-is-a-new-line-at-end-of-file